src/main/java/dev/ai4j/openai4j/OpenAiClient.java

```diff
@@ -67,6 +67,8 @@ public abstract static class Builder<T extends OpenAiClient, B extends Builder<T
         public Proxy proxy;
         public boolean logRequests;
         public boolean logResponses;
+        public LogLevelEnum logRequestsLevel;
+        public LogLevelEnum logResponsesLevel;
```

Comment:
Would be a single setting `logLevel` just not enough? 🤔

Reply:
For my specific case, yes. I added both to match the "logRequests" and "logResponses". Want to remove one of them and use only one for request and response?

Reply:
exactly, one setting for both, I guess it should be enough